# 04 COT positioning

After managed money net on WTI-PHYSICAL is extremely long versus the pre-2020 book, is the next five-print RWTC return higher?

Holdout starts 2020-01-01. Verdict: failed.

This study reads stored series. It does not add a derived column.

In [1]:
import json
from pathlib import Path
studies = json.loads(Path('data/studies.json').read_text())
study = next(item for item in studies['studies'] if item['slug'] == 'positioning')
print(study['verdict'])
print(study['verdict_line'])
print(study['table']['caption'])
for row in study['table']['rows']:
    print(' | '.join(row))


failed
The frozen 90th percentile of managed-money net over open interest is 0.1653. Train five-print return above that line minus the return below it is -0.444%. Holdout difference is 1.034%. The holdout flipped the train sign.
Five-print RWTC return after CFTC_CL managed-money net / open interest
Train | At or above 0.1653 | 52 | -0.471%
Train | Below the frozen line | 467 | -0.027%
Holdout | At or above 0.1653 | 16 | 1.023%
Holdout | Below the frozen line | 335 | -0.011%


## Method

The 90th percentile is the nearest rank of the train ratios only. The holdout is cut with that same number.

## Result

The frozen 90th percentile of managed-money net over open interest is 0.1653. Train five-print return above that line minus the return below it is -0.444%. Holdout difference is 1.034%. The holdout flipped the train sign.